app/views/home_page/_header.html.erb

```diff
@@ -2,8 +2,7 @@
   <div class="mx-auto max-w-md px-4 sm:max-w-3xl sm:px-6 lg:px-8 lg:max-w-7xl lg:grid lg:grid-cols-2 lg:gap-24">
     <div>
       <div class="flex items-center">
-        <%= render "icons/fire", classes: "h-14 w-14" %>
-        <h1 class="ml-1 text-3xl font-extrabold hidden lg:block">hotseat.io</h1>
+        <%= render "icons/hotseatlogotext_red", classes: "absolute w-522 h-150 left-101 top-340" %>
```

Comment:
Can we avoid using `absolute` positioning here? Happy to pair on this with you if helpful!